src/scss/components/_navbar.scss

```diff
@@ -120,10 +127,6 @@ $header-height: 10;
       .usa-current {
         @include add-bar(0.5, 'secondary', 'bottom', 0, 1.5, 0);
         font-weight: font-weight('bold');
-
-        &:hover {
-          color: color('primary-dark');
-        }
```

Comment:
Not strictly related to the changes, but these styles are redundant with those above:

https://github.com/18F/identity-style-guide/blob/main/src/scss/components/_navbar.scss#L109-L112